design/tls-client-verification.md

```diff
@@ -0,0 +1,125 @@
+# External client certificate validation (mutual TLS authentication)
+
+Status: Draft
+
+This document outlines how to add support for authentication of external
```

Comment:
Do not wrap lines. One line per sentence please. This makes it easier to comment a specific line, and reduces the diff size for comments.

Reply:
I have now reformatted the document accordingly.